postman/Get Broadcast Details.postman_collection.json

```diff
@@ -0,0 +1,549 @@
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=draft",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts",
+						"total"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "draft"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Broadcast Totals  - Scheduled",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "aebe59b2-ac5a-41af-9f56-21bd56794e16",
+						"exec": [
+							"pm.test(\"Successfully retrieved the total number of scheduled broadcasts on list\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.a('number')",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=scheduled",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts",
+						"total"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "scheduled"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Broadcast Totals  - Sent",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "96754c67-4f3b-4523-9f57-27cf07835971",
+						"exec": [
+							"pm.test(\"Successfully retrieved the total number of sent broadcasts on list\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.a('number')",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=sent",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts",
+						"total"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "sent"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Draft Broadcasts",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "0d1babcb-7bac-46f3-8982-dd5d013f4b79",
+						"exec": [
+							"pm.test(\"Successfully retrieved link for first draft broadcast\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    if (pm.expect(jsonData.entries.length).is.above(0)) {",
```

Comment:
You don't need to check the return value here. The assertion will stop the test if it fails.

Reply:
This is still checking the return value